### PR TITLE
Created README template for future update

### PR DESCRIPTION
**Pull request by homa31**
#### Issue Description
Brushed up README.md

#### Additional Info
Anything I should add or change?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+## Description
+[Babble](https://babble-lilac.vercel.app/) provides live video chat and text field services. It enables people from around the world to work together. Users can create rooms with passwords, invite people to their rooms, live video chat and type in a shared text field.
+
+## Demo
+Live demo available here: https://babble-lilac.vercel.app/
+
+## Site
+
+### Landing Page
+![image of babble landing page]()
```
Comment (homa31) [inline README.md:24]: Will add pictures after design is finalized

**Review by bootsie123 — APPROVED** [review]
Looks good!

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+### Rooms
+![image of babble rooms page]()
+
+## Technologies
+* Vue.js with Tailwind CSS
+* [Zoom SDK](https://marketplace.zoom.us/docs/sdk/native-sdks/introduction)
```
Comment (jake-pauls) [inline README.md]: Is it cool if this gets changed to be [PeerJS](https://peerjs.com/)? That's what I'm working on instead of the Zoom SDK at the moment!

**Review by jake-pauls — COMMENTED** [review]
Looks really great! Once the change to the Zoom SDK is in it's good to go :+1:

**Pull request merged**